src/components/KCodeBlock/KCodeBlock.vue

```diff
@@ -897,10 +896,9 @@ $kCodeBlockDarkLineMatchBackgroundColor: rgba(255, 255, 255, 0.12); // we don't
 
     .code-block-secondary-actions {
       display: flex;
+      float: right;
```

Comment:
We tend to avoid using `float`.

Reply:
+1

`float` will cause the code area to shrink:

<img width="993" alt="image" src="https://github.com/user-attachments/assets/01b29815-4574-4a70-a056-58b859a7062f" />

Reply:
Added the changes, changed the structure a bit, by nesting the code-content and also added a sticky wrapper.